src/main/webapp/view.js

```diff
@@ -15,22 +15,53 @@
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
 
+/** Timestamp that video, disccussion, and transcript were synced to. */
+let lastSyncedTime;
+
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
 initialize();
 
 /**
- * Initializes the video player and discussion section
+ * Initializes the timer, video player, and discussion section
  * for the lecture view page.
  */
 async function initialize() {
   window.loadVideoApi();
-  window.intializeDiscussion();
+  window.loadDiscussion();
+  // TODO: Adjust time interval.
+  window.timer = window.setInterval(syncEverythingToVideo, 1000);
   // TODO: Initialize the trancript section.
 }
 
+/**
+ * Retrives the current time of the video and relays time to
+ * {@code sync}.
+ */
+function syncEverythingToVideo() {
```

Comment:
Do you think you can combine `syncEverythingToVideo` with `sync`? I find it redundant that there's two methods for syncing. Since there's already a boolean for the video in `sync`, I think you can have a check at the beginning to see if its the video and update the current time there.

Reply:
I agree, another option to consider is in-lining it in the `setInterval`

Reply:
made inline